import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,55 @@
+
+import { GetObjectCommand, CopyObjectCommand, DeleteObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+
+const csv = require('csv-parser')
+
+const importProductsFile = async (event) => {
+  try{
+    console.log('event', event);
+
+    const s3Client = new S3Client({region: 'us-east-1'});
+    const bucket = 'aws-uploaded-bucket';
+
+    for (const record of event.Records) {
+      const getCommand = new GetObjectCommand({
+          Bucket: bucket,
+          Key: `uploaded/${record.s3.object.key}`,
+      })
+
+			const stream = await s3Client.send(getCommand);
+
+			await new Promise((resolve, reject) => {
```

Comment:
nice workaround to this point 

- [ ] At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into a new folder in the same bucket called parsed, and then deleted from uploaded folder)

I will probably steal it, lol